pkgs/development/compilers/dotnet/update.sh

```diff
@@ -252,7 +252,8 @@ sdk_packages () {
       "Microsoft.NETCore.App.Crossgen2.linux-arm64" \
       "Microsoft.NETCore.App.Crossgen2.linux-x64" \
       "Microsoft.NETCore.App.Crossgen2.osx-x64" \
-      "Microsoft.NETCore.App.Crossgen2.osx-arm64"
+      "Microsoft.NETCore.App.Crossgen2.osx-arm64" \
+      "Microsoft.NET.ILLink.Tasks"
```

Comment:
I think you need to make this version >= 8.  I did something similar here:

https://github.com/NixOS/nixpkgs/pull/190129/commits/b37cdccf9bcbd6bbd0fdc3f00883849e432f831c

But it probably won't get merged before this PR.

Reply:
Error handling is currently a bit broken, so I'm not sure what will actually happen.

This is my attempt to fix the error handling: https://github.com/NixOS/nixpkgs/pull/190129/commits/6d9273e352bb0936db32784a292aa4201d335e4c

Reply:
This doesn't actually break anything on 6 because it's (currently) the last package.  However on 7, all the `ILCompiler` packages are missing.

Reply:
I made a PR with my update script changes: #290351